Provider/src/main/java/com/spotify/confidence/openfeature/ConfidenceFeatureProvider.kt

```diff
@@ -143,6 +148,23 @@ class ConfidenceFeatureProvider private constructor(
     companion object {
         private class ConfidenceMetadata(override var name: String? = "confidence") : ProviderMetadata
 
+        fun createConfidence(
+            context: Context,
+            clientSecret: String,
+            initialContext: Map<String, ConfidenceValue> = mapOf(),
+            region: ConfidenceRegion = ConfidenceRegion.GLOBAL,
+            dispatcher: CoroutineDispatcher = Dispatchers.IO
+        ): Confidence {
```

Comment:
I am wondering if we could introduce a new interface `ConfidenceForOpenFeature` or something, and make the `create` function below accept that, as a way to force users to use this `createConfidence`

Reply:
in this case `ConfidenceForOpenFeature ` should inherit both from event sender and from `Contextual`

Reply:
and all other functions are from confidence object itself, we don't have a confidence interface currently.

Reply:
Perhaps worth exploring, but I am also not strongly convinced these extra interfaces will actually help the user

Reply:
do you agree taking this in another PR and moving forward with this PR?